### DISCUSSION TITLE
浅谈Nya LoCyanFrp!开发 - 夏沫花火zzz🌙 (Muska_Ami)のLife

### DISCUSSION THREAD

**Discussion by giscus** [Muska-Ami/muska-ami-blog#75] [General]
# 浅谈Nya LoCyanFrp!开发 - 夏沫花火zzz🌙 (Muska_Ami)のLife

什么是 Nya LoCyanFrp!Nya LoCyanFrp! (简写 NyaLCF) 是由本人开发的 LoCyanFrp 映射客户端，原为非官方客户端，现由于我加入了 LCT 团队（LoCyanTeam），所以转为官方客户端。GitHub: https://github.com/Muska-Ami/NyaLCF  开发这款启动器的原因如果你和我一样是从一代 LCF 开始使用的，那么你应该知道原

https://blog.1l1.icu/2024/06/03/qian-tan-nya-locyanfrp-kai-fa/

**Comment by ltzXiaoYanMo** [comment]
合作者在此打卡！
